### ISSUE TITLE
Cannot read properties of undefined (reading 'join')

### ISSUE THREAD

**Issue by alefyp** [betrueagency/nx-reactjs-elementor#8]
Hello there 

I'm getting this error while executing `npx nx g @betrue/react-elementor:plugin project-name`

```
✔ Which stylesheet format would you like to use? · scss
my-project-input
my-project-title,my-project-input

 >  NX   Cannot read properties of undefined (reading 'join')
```

Any idea what could be causing this error?  

The verbose option log this files:

```
TypeError: Cannot read properties of undefined (reading 'join')
    at .../my-workspace/node_modules/@betrue/react-elementor/src/generators/plugin/lib/react-component-files.js:76:94
    at step (.../my-workspace/node_modules/@betrue/react-elementor/src/generators/plugin/lib/react-component-files.js:44:23)
    at Object.next (.../my-workspace/node_modules/@betrue/react-elementor/src/generators/plugin/lib/react-component-files.js:25:53)
    at fulfilled (.../my-workspace/node_modules/@betrue/react-elementor/src/generators/plugin/lib/react-component-files.js:16:58)
```
<img width="990" alt="Screen Shot 2022-10-25 at 12 08 18 PM" src="https://user-images.githubusercontent.com/74845080/197838191-f0f61f02-7be8-4abf-8590-8eb9a967bcad.png">

**Comment by alizarion** [comment]
hi @alefyp 
this was a typescript compilation issue that i have fix in the release v1.0.3 
please retry and let me now if it work
note: for shadowdom css wrapping reason, the plugin work actually only with styled-component

**Comment by alefyp** [comment]
Up and running! it was that little detail about styled components, I see you guys updated the documentation as well, thank you very much :)
